ext/fg/js/float.js

```diff
@@ -45,6 +45,8 @@ class DisplayFloat extends Display {
         }
 
         this.setContentScale(scale);
+
+        window.parent.postMessage('initialized', '*');
```

Comment:
Indicating to the popup class that the iframe is fully read is a good idea, but there are probably a few issues with this implementation:

1. Multiple popups can be on the same page due to the nested popup and proxy systems. A unique ID may need to be generated for each popup to make sure the messages are handled by the appropriate location.

2. The message is being posted to the root HTML page. This means that it is not being sent exclusively to the extension, and that it could affect the page the script is running on. For example, if a website is has a handler for an 'initialized' method, this could interfere with its operation.

I haven't tested these claims myself, but the `targetOrigin` being `'*'` likely has the same concerns as what was noted in #353. It may be possible to just have the float frame post a message directly to its tab using [`sendMessage`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/sendMessage) to avoid the concerns with `postMessage`. Still will probably need a unique identifier.

Reply:
> It may be possible to just have the float frame post a message directly to its tab using [`sendMessage`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/sendMessage) to avoid the concerns with `postMessage`. Still will probably need a unique identifier.

I actually tested `apiForward` that uses this but it only worked for the first popup. Should have probably added the method to `PopupProxy` and `PopupProxyHost` too. They use some sort of ID, maybe that handles sending the message to the correct popup?

Reply:
Yeah, the existing id of the popup/proxy popup should work.

Reply:
Should be fixed in https://github.com/FooSoft/yomichan/pull/357/commits/810a7e7d92d15412974810702d954de60453dd31